From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Dmitry <docent27@ukr.net>
Date: Sat, 10 Jul 2021 12:06:25 +0300
Subject: [PATCH] nogui

---
 apps/openmw/mwgui/windowmanagerimp.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/apps/openmw/mwgui/windowmanagerimp.cpp b/apps/openmw/mwgui/windowmanagerimp.cpp
index 544a092..cf8058d 100644
--- a/apps/openmw/mwgui/windowmanagerimp.cpp
+++ b/apps/openmw/mwgui/windowmanagerimp.cpp
@@ -188,7 +188,7 @@ namespace MWGui
     {
         mScalingFactor = std::clamp(Settings::Manager::getFloat("scaling factor", "GUI"), 0.5f, 8.f);
         mGuiPlatform = new osgMyGUI::Platform(viewer, guiRoot, resourceSystem->getImageManager(), mScalingFactor);
-        mGuiPlatform->initialise(resourcePath, (boost::filesystem::path(logpath) / "MyGUI.log").generic_string());
+        mGuiPlatform->initialise(resourcePath, logpath);
 
         mGui = new MyGUI::Gui;
         mGui->initialise("");
--
libgit2 0.28.4
