diff --git a/boost/libs/filesystem/src/operations.cpp b/boost/libs/filesystem/src/operations.cpp
index 4114e02a..e663fabc 100644
--- a/boost/libs/filesystem/src/operations.cpp
+++ b/boost/libs/filesystem/src/operations.cpp
@@ -2169,8 +2169,6 @@ namespace
   inline int readdir_r_simulator(DIR * dirp, struct dirent * entry,
     struct dirent ** result)// *result set to 0 on end of directory
   {
-    errno = 0;
-
 #   if !defined(__CYGWIN__)\
     && defined(_POSIX_THREAD_SAFE_FUNCTIONS)\
     && defined(_SC_THREAD_SAFE_FUNCTIONS)\
@@ -2183,6 +2181,7 @@ namespace
 
     struct dirent * p;
     *result = 0;
+    errno = 0;
     if ((p = ::readdir(dirp))== 0)
       return errno;
     std::strcpy(entry->d_name, p->d_name);