diff --git a/src/gl/texture_compressed.c b/src/gl/texture_compressed.c
index 4101519..6b8c8f8 100755
--- a/src/gl/texture_compressed.c
+++ b/src/gl/texture_compressed.c
@@ -214,11 +214,11 @@ void gl4es_glCompressedTexImage2D(GLenum target, GLint level, GLenum internalfor
                 #endif
             }
         }
-        if(level && bound->valid) {
+/*        if(level && bound->valid) {
             // don't mix type/format along mipmap...
             format = bound->format;
             type = bound->type;
-        }
+        }*/
         int srgb = isDXTcSRGB(internalformat);
         int simpleAlpha = 0;
         int complexAlpha = 0;
