diff --git a/Android.mk b/Android.mk
index 88b53fd..99a03fc 100755
--- a/Android.mk
+++ b/Android.mk
@@ -19,6 +19,7 @@ LOCAL_SRC_FILES := \
 	src/gl/blend.c \
 	src/gl/blit.c \
 	src/gl/buffers.c \
+	src/gl/build_info.c \
 	src/gl/debug.c \
 	src/gl/decompress.c \
 	src/gl/depth.c \
@@ -32,7 +33,7 @@ LOCAL_SRC_FILES := \
 	src/gl/fpe_shader.c \
 	src/gl/framebuffers.c \
 	src/gl/getter.c \
-	src/gl/gl.c \
+	src/gl/gl4es.c \
 	src/gl/glstate.c \
 	src/gl/hint.c \
 	src/gl/init.c \
@@ -63,7 +64,7 @@ LOCAL_SRC_FILES := \
 	src/gl/texture.c \
 	src/gl/uniform.c \
 	src/gl/vertexattrib.c \
-	src/gl/wrap/gl.c \
+	src/gl/wrap/gl4es.c \
 	src/gl/wrap/gles.c \
 	src/gl/wrap/glstub.c \
 	src/gl/math/eval.c \
diff --git a/src/glx/lookup.c b/src/glx/lookup.c
index 85d3807..f90af1b 100755
--- a/src/glx/lookup.c
+++ b/src/glx/lookup.c
@@ -1,5 +1,5 @@
 #ifdef ANDROID
-#include "../gl/gl.h"
+#include "../gl/gl4es.h"
 #endif
 #include "glx.h"
 #include "hardext.h"
