diff --git a/components/sceneutil/shadow.cpp b/components/sceneutil/shadow.cpp
index a69fd8090..aa94e14d0 100644
--- a/components/sceneutil/shadow.cpp
+++ b/components/sceneutil/shadow.cpp
@@ -57,6 +57,7 @@ namespace SceneUtil
 
     void ShadowManager::disableShadowsForStateSet(osg::ref_ptr<osg::StateSet> stateset)
     {
+        return; // fixes red sky/minimap/etc
         int numberOfShadowMapsPerLight = Settings::Manager::getInt("number of shadow maps", "Shadows");
         int baseShadowTextureUnit = 8 - numberOfShadowMapsPerLight;
         
