From 1458853560e6f05a5b8935eff83133471d493e42 Mon Sep 17 00:00:00 2001
From: Ilya Zhuravlev <whatever@xyz.is>
Date: Sun, 10 Feb 2019 19:29:48 -0500
Subject: [PATCH 3/5] Array: make possible to use thread-safely with
 setTexCoordArray

---
 include/osg/Array | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/include/osg/Array b/include/osg/Array
index 7155ddeee..6480c7949 100644
--- a/include/osg/Array
+++ b/include/osg/Array
@@ -186,7 +186,11 @@ class OSG_EXPORT Array : public BufferData
 
 
         /** Specify how this array should be passed to OpenGL.*/
-        void setBinding(Binding binding) { _binding = binding; }
+        void setBinding(Binding binding) {
+            // N.B. race if not initialized before being shared between different threads
+            if (_binding != binding)
+                _binding = binding;
+        }
 
         /** Get how this array should be passed to OpenGL.*/
         Binding getBinding() const { return _binding; }
-- 
2.19.2

