From 8c70bbf13ebe03ae333b6de67201e4d4c163017c Mon Sep 17 00:00:00 2001
From: Ilya Zhuravlev <whatever@xyz.is>
Date: Sun, 10 Feb 2019 19:12:00 -0500
Subject: [PATCH 2/8] scenemanager: disable state share / inc compile

---
 components/resource/scenemanager.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/components/resource/scenemanager.cpp b/components/resource/scenemanager.cpp
index 61a40ee4b..dda385c73 100644
--- a/components/resource/scenemanager.cpp
+++ b/components/resource/scenemanager.cpp
@@ -515,9 +515,11 @@ namespace Resource
             // share state
             // do this before optimizing so the optimizer will be able to combine nodes more aggressively
             // note, because StateSets will be shared at this point, StateSets can not be modified inside the optimizer
+            #if 0
             mSharedStateMutex.lock();
             mSharedStateManager->share(loaded.get());
             mSharedStateMutex.unlock();
+            #endif
 
             if (canOptimize(normalized))
             {
@@ -529,9 +531,11 @@ namespace Resource
                 optimizer.optimize(loaded, options);
             }
 
+            #if 0
             if (mIncrementalCompileOperation)
                 mIncrementalCompileOperation->add(loaded);
             else
+            #endif
                 loaded->getBound();
 
             mCache->addEntryToObjectCache(normalized, loaded);
-- 
2.19.2

