From f5c92393f41d6873ea34245207dd28466e6ebf13 Mon Sep 17 00:00:00 2001
From: Ilya Zhuravlev <whatever@xyz.is>
Date: Sun, 10 Feb 2019 19:12:00 -0500
Subject: [PATCH 2/4] scenemanager: disable state share / inc compile

---
 components/resource/scenemanager.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/components/resource/scenemanager.cpp b/components/resource/scenemanager.cpp
index 328a10cd1..ee68751a3 100644
--- a/components/resource/scenemanager.cpp
+++ b/components/resource/scenemanager.cpp
@@ -526,9 +526,11 @@ namespace Resource
             // share state
             // do this before optimizing so the optimizer will be able to combine nodes more aggressively
             // note, because StateSets will be shared at this point, StateSets can not be modified inside the optimizer
+            #if 0
             mSharedStateMutex.lock();
             mSharedStateManager->share(loaded.get());
             mSharedStateMutex.unlock();
+            #endif
 
             if (canOptimize(normalized))
             {
@@ -540,8 +542,10 @@ namespace Resource
                 optimizer.optimize(loaded, options);
             }
 
+            #if 0
             if (mIncrementalCompileOperation)
                 mIncrementalCompileOperation->add(loaded);
+            #endif
 
             mCache->addEntryToObjectCache(normalized, loaded);
             return loaded;
-- 
2.19.2

