From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ilya Zhuravlev <whatever@xyz.is>
Date: Mon, 1 Apr 2019 13:32:32 -0400
Subject: [PATCH] android: Revert UI dimensions

---
 files/settings-default.cfg | 95 ++++++++++++++++++++------------------
 1 file changed, 49 insertions(+), 46 deletions(-)

diff --git a/files/settings-default.cfg b/files/settings-default.cfg
index fb5862fad..1675261d5 100644
--- a/files/settings-default.cfg
+++ b/files/settings-default.cfg
@@ -482,42 +482,45 @@ refraction scale = 1.0
 # screen size. (0.0 to 1.0).  X & Y, Width & Height.
 
 # Stats window displaying level, race, class, skills and stats.
-stats x = 0.015
-stats y = 0.015
-stats w = 0.4275
-stats h = 0.45
+stats x = 0.0
+stats y = 0.0
+stats w = 0.375
+stats h = 0.4275
+# Stats window pin status
 stats pin = false
 stats hidden = false
 
 # Spells window displaying powers, spells, and magical items.
-spells x = 0.63
-spells y = 0.39
-spells w = 0.36
-spells h = 0.51
+spells x = 0.625
+spells y = 0.5725
+spells w = 0.375
+spells h = 0.4275
+# Spells window pin status
 spells pin = false
 spells hidden = false
 
 # Local and world map window.
-map x = 0.63
-map y = 0.015
-map w = 0.36
-map h = 0.37
+map x = 0.625
+map y = 0.0
+map w = 0.375
+map h = 0.5725
+# Map window pin status
 map pin = false
 map hidden = false
 
 # Player inventory window when explicitly opened.
-inventory x = 0.015
-inventory y = 0.54
-inventory w = 0.45
-inventory h = 0.38
+inventory x = 0.0
+inventory y = 0.4275
+inventory w = 0.6225
+inventory h = 0.5725
 inventory pin = false
 inventory hidden = false
 
 # Dialog window for talking with NPCs.
-dialogue x = 0.15
-dialogue y = 0.5
-dialogue w = 0.7
-dialogue h = 0.45
+dialogue x = 0.095
+dialogue y = 0.095
+dialogue w = 0.810
+dialogue h = 0.810
 
 # Alchemy window for crafting potions.
 alchemy x = 0.25
@@ -526,46 +529,46 @@ alchemy w = 0.5
 alchemy h = 0.5
 
 # Console command window for debugging commands.
-console x = 0.015
-console y = 0.015
+console x = 0.0
+console y = 0.0
 console w = 1.0
 console h = 0.5
 
 # Player inventory window when searching a container.
-inventory container x = 0.015
-inventory container y = 0.54
-inventory container w = 0.45
-inventory container h = 0.38
+inventory container x = 0.0
+inventory container y = 0.4275
+inventory container w = 0.6225
+inventory container h = 0.5725
 
 # Player inventory window when bartering with a shopkeeper.
-inventory barter x = 0.015
-inventory barter y = 0.54
-inventory barter w = 0.45
-inventory barter h = 0.38
+inventory barter x = 0.0
+inventory barter y = 0.4275
+inventory barter w = 0.6225
+inventory barter h = 0.5725
 
 # Player inventory window when trading with a companion.
-inventory companion x = 0.015
-inventory companion y = 0.54
-inventory companion w = 0.45
-inventory companion h = 0.38
+inventory companion x = 0.0
+inventory companion y = 0.4275
+inventory companion w = 0.6225
+inventory companion h = 0.5725
 
 # Container inventory when searching a container.
-container x = 0.49
-container y = 0.54
-container w = 0.39
-container h = 0.38
+container x = 0.25
+container y = 0.0
+container w = 0.75
+container h = 0.375
 
 # NPC inventory window when bartering with a shopkeeper.
-barter x = 0.6
-barter y = 0.27
-barter w = 0.38
-barter h = 0.63
+barter x = 0.25
+barter y = 0.0
+barter w = 0.75
+barter h = 0.375
 
 # NPC inventory window when trading with a companion.
-companion x = 0.6
-companion y = 0.27
-companion w = 0.38
-companion h = 0.63
+companion x = 0.25
+companion y = 0.0
+companion w = 0.75
+companion h = 0.375
 
 [Navigator]
 
-- 
2.19.2

