From e3c3ca0a06fa6695259054850c13159e6fe60d77 Mon Sep 17 00:00:00 2001
From: Ilya Zhuravlev <whatever@xyz.is>
Date: Mon, 1 Apr 2019 13:33:12 -0400
Subject: [PATCH 7/9] android: Fix red sky after shadows merge

---
 components/sceneutil/shadow.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/components/sceneutil/shadow.cpp b/components/sceneutil/shadow.cpp
index a69fd8090..aa94e14d0 100644
--- a/components/sceneutil/shadow.cpp
+++ b/components/sceneutil/shadow.cpp
@@ -57,6 +57,7 @@ namespace SceneUtil
 
     void ShadowManager::disableShadowsForStateSet(osg::ref_ptr<osg::StateSet> stateset)
     {
+        return; // fixes red sky/minimap/etc
         int numberOfShadowMapsPerLight = Settings::Manager::getInt("number of shadow maps", "Shadows");
         int baseShadowTextureUnit = 8 - numberOfShadowMapsPerLight;
         
-- 
2.19.2

