From 80fb596ce589437e6a5c3f8a6520f3c5a1d0aee5 Mon Sep 17 00:00:00 2001
From: Ilya Zhuravlev <whatever@xyz.is>
Date: Mon, 29 Apr 2019 18:27:40 -0400
Subject: [PATCH 12/13] engine: pause sounds when game is paused

---
 apps/openmw/engine.cpp | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/apps/openmw/engine.cpp b/apps/openmw/engine.cpp
index ed2a0422a..ec29e896a 100644
--- a/apps/openmw/engine.cpp
+++ b/apps/openmw/engine.cpp
@@ -93,6 +93,22 @@ bool OMW::Engine::frame(float frametime)
         // update input
         mEnvironment.getInputManager()->update(frametime, false);
 
+        // Android(start)
+        bool pause_new = !mEnvironment.getInputManager()->isWindowVisible();
+        static bool pause_prev = pause_new; // n.b. this is only set once, not on every call to frame()
+
+        if (pause_prev != pause_new) {
+            if (pause_new) {
+                // if we went from unpaused=>paused, stop all sounds
+                mEnvironment.getSoundManager()->pauseSounds();
+            } else {
+                // if we went from paused=>unpaused, resume all sounds
+                mEnvironment.getSoundManager()->resumeSounds();
+            }
+        }
+        pause_prev = pause_new;
+        // Android(end)
+
         // When the window is minimized, pause the game. Currently this *has* to be here to work around a MyGUI bug.
         // If we are not currently rendering, then RenderItems will not be reused resulting in a memory leak upon changing widget textures (fixed in MyGUI 3.3.2),
         // and destroyed widgets will not be deleted (not fixed yet, https://github.com/MyGUI/mygui/issues/21)
-- 
2.19.2

