From aab6f1f219723f36eb6d3a47f390964823bbfd7b Mon Sep 17 00:00:00 2001
From: Ilya Zhuravlev <whatever@xyz.is>
Date: Thu, 2 May 2019 11:15:52 -0400
Subject: [PATCH 13/13] android: Update default settings for android

---
 files/settings-default.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/settings-default.cfg b/files/settings-default.cfg
index be699ed32..818b8781c 100644
--- a/files/settings-default.cfg
+++ b/files/settings-default.cfg
@@ -23,7 +23,7 @@ small feature culling pixel size = 2.0
 
 # Maximum visible distance (e.g. 2000.0 to 6666.0).  Caution: this setting
 # can dramatically affect performance, see documentation for details.
-viewing distance = 6666.0
+viewing distance = 2000.0
 
 # Camera field of view in degrees (e.g. 30.0 to 110.0).
 # Does not affect the player's hands in the first person camera.
-- 
2.19.2

